store/evmstore.go

```diff
@@ -236,23 +190,11 @@ func (s *EvmStore) Commit(version int64) []byte {
 	}
 
 	s.rootCache.Add(version, currentRoot)
-
-	batch := s.evmDB.NewBatch()
-	for key, item := range s.cache {
-		if !item.Deleted {
-			batch.Set([]byte(key), item.Value)
-		} else {
-			batch.Delete([]byte(key))
-		}
-	}
-	batch.Write()
-	s.cache = make(map[string]cacheItem)
```

Comment:
Hmm, so no cache is fine, but no batching less so. Should still batch the writes to the DB.

Reply:
We only write to evmstore in `TrieDB.Commit()` which already uses ethDB.batch so it is fine. https://github.com/loomnetwork/go-ethereum/blob/6128fa1a8c767035d3da6ef0c27ebb7778ce3713/trie/database.go#L620

Reply:
You're right, there is already batching in `LoomEthDB`, but it's crap because each key in the batch is written out individually to `evm.db` by the `EvmStore`. Should make `EvmStore` expose a batch API so `LoomEthDB` can batch writes properly.